api/data_explorer/util/elasticsearch_util.py

```diff
@@ -193,7 +193,43 @@ def get_field_type(es, field_name):
         field_name]['mapping'][last_part]['type']
 
 
-def get_elasticsearch_facet(es, elasticsearch_field_name, field_type):
+def _get_nested_field_paths(prefix, mappings):
```

Comment:
_get_nested_paths_inner?

(It's not clear what the difference between `get_nested_paths()` and `_get_nested_field_paths()` is.)